app/models/track.rb

```diff
@@ -100,6 +102,25 @@ def to_param
     short_name
   end
 
+  def user_rating(user)
```

Comment:
These methods are used in both Track and Event models. Should we extract them to avoid duplication?